languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -28745,4 +28607,609 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+		        <example correction="Um sich umzuziehen," type="incorrect"><marker>Um sich umzuziehen</marker> ging sie nach Hause.</example>
+		        <example><marker>Um sich umzuziehen,</marker> ging sie nach Hause.</example>
+		    </rule>
+	    </rulegroup>
+        <rulegroup id="KOMMAFEHLER" name="Fehler in der Kommasetzung">   
+		    <url>http://www.duden.de/sprachwissen/rechtschreibregeln/komma</url>
+		    <rule id="KOMMA_VOR_SONDERN" name="Komma vor 'sondern'">
+		        <pattern>
+					<marker>
+		            <token><exception regexp="yes">,|;|-|–|\(|„|“</exception></token>
+					<token>sondern</token>
+					</marker>
+		        </pattern>
+		        <message>Bitte ein Komma vor '\2' setzen!</message>
+	            <suggestion>\1, \2</suggestion>
+		        <example correction="Sommer, sondern" type="incorrect">Es ist nicht <marker>Sommer sondern</marker> Winter.</example>
+		        <example>Es ist nicht <marker>Sommer, sondern</marker> Winter.</example>
+		    </rule>
+		    <rule id="KOMMA_VOR_ERLAEUTERUNG" name="Komma vor Erläuterung" default="off">
+<!-- Sonderfälle noch nicht behandelt, deshalb deafult="off"  -->
+		        <pattern>
+					<marker>
+		            <token><exception regexp="yes">,|;|-|–|\(|„|“</exception></token>
+					<token regexp="yes">nämlich|insbesondere</token>
+					</marker>
+		        </pattern>
+		        <message>Bitte ein Komma vor '\2' setzen!</message>
+	            <suggestion>\1, \2</suggestion>
+		        <example correction="Jahreszeiten, nämlich" type="incorrect">Es gibt vier <marker>Jahreszeiten nämlich</marker> Frühling, Sommer, Herbst und Winter.</example>
+		        <example>Es gibt vier <marker>Jahreszeiten, nämlich</marker> Frühling, Sommer, Herbst und Winter.</example>
+		    </rule>
+		    <rule id="KOMMA_VOR_UND_ODER" name="Fehlerhaftes Komma vor 'und' oder 'oder'">
+		        <antipattern>
+					<marker>
+					<token skip="-1">,</token>
+					<token>,</token>
+					<token regexp="yes">und|oder</token>
+					</marker>
+		        </antipattern>
+		        <antipattern>
+					<marker>
+					<token>,</token>
+					<token>und</token>
+					<token>zwar</token>
+					</marker>
+		        </antipattern>
+		        <pattern>
+					<marker>
+					<token>,</token>
+					<token regexp="yes">und|oder</token>
+					</marker>
+		        </pattern>
+		        <message>Bitte das Komma vor und/oder löschen!</message>
+	            <suggestion> \2</suggestion>
+		        <example correction=" und" type="incorrect">Die Sonne versank hinter dem Horizont<marker>, und</marker> die Schatten der Nacht senkten sich über das Land.</example>
+		        <example>Die Sonne versank hinter dem Horizont<marker> und</marker> die Schatten der Nacht senkten sich über das Land.</example>
+		    </rule>
+		    <rule id="KOMMA_NACH_DIREKTER_REDE" name="Fehlendes Komma nach direkter Rede">
+		        <antipattern>
+					<token postag="SENT_START"/>
+					<token regexp="yes">»|«|"</token>
```

Comment:
Hier noch eine weiteres Beispiel:
"Es ist besser etwas einmal zu sehen als zehnmal davon zu hören." Die Regel schlägt ein Komma vor 'als' vor.
Der Duden sagt dazu unter "Kommasetzung bei Vergleichen":
"Auch Infinitivgruppen bzw. Infinitive mit zu, die mit als eingeleitet sind, werden durch Komma abgetrennt: Es gibt für mich nichts Schöneres, als am Sonntagmorgen gemütlich im Bett zu frühstücken. Er konnte nichts Besseres tun, als zu reisen. Dagegen ohne Komma: Er konnte nichts Besseres tun als reisen."
Ich fürchte, in den Tatoeba Beispielsätzen befinden sich Kommafehler.